Readme.md

```diff
@@ -173,6 +173,32 @@ AppConfig[:aeon_fulfillment] = {
   this setting is not specified in the settings for the repository, no Aeon
   site code will be sent.
 
+- **:hide\_button\_for\_access\_restriction\_types**. This setting allows
+  the request button to be hidden for any records that have any of the
+  listed local access restriction types. The value of this config item
+  should be an array of restriction types, for example:
+      `:hide_button_for_access_restriction_types => ['RestrictedSpecColl']`
+  By default no restriction types are hidden.
+
+- **:site**. This setting specifies the site code for a repository.
+
+
+### Other Configuration Options
+
+The following configuration options apply globally, rather than for a particular
+repository.
+
+- **:aeon\_fulfillment\_record\_types**. This setting takes an array of record
+  types. It allows this plugin to handle additional record types via custom
```

Comment:
Thank you for including documentation in this pull request. I can clean up some things after it is submitted, including the table of contents and the changelog. Would you like me to add a link from this setting to the Custom Mappers section below?

Reply:
That sounds good - thanks